### ISSUE TITLE
Caret is not displayed

### ISSUE THREAD

**Issue by PumpkinSeed** [iafan/goplayspace#9]
I really miss the cursor.

**Comment by iafan** [comment]
I guess by cursor you mean the blinking caret in the textarea? Can you please provide more information (your OS and browser version)?

**Comment by PumpkinSeed** [comment]
Yes, but I'm sorry I thought it's not there because of conscious implementation. I'm on Fedora 25 and Chomre 57.0.2987.133 (64-bit).
Thank you.

**Comment by iafan** [comment]
I wouldn't expect any problems in modern Chrome, but this might be OS-specific somehow. Wish someone having Fedora could confirm the issue. Is that the latest version of Chrome on Fedora? On Mac I'm having version 58, so I'd suggest to update and restart the browser just in case.

Can you try a different theme (click the `Settings` button and then select a theme from a drop-down list) and see if the caret is invisible in any theme or just specific ones?

Also, could you try this in Firefox as well to see if this bug is browser-specific or OS-specific?

**Comment by PumpkinSeed** [comment]
I gave a try to all theme, but I got the same error. It is the latest Chrome for Fedora.

**Comment by xi2** [comment]
Just as an additional data point, I also experience the missing caret.

Browser: Firefox ESR 45.9.0 (latest Debian version).

OS: Debian GNU/Linux.

**Comment by iafan** [comment]
Thanks for keeping the feedback coming, folks. Until I'm able to reproduce this issue, I'm planning to adjust the code so that it works as a vanilla textarea (but with line numbers) in a non-syntax-highlighted mode. Having no syntax highlighting is probably a lesser problem than having no caret at all.

**Comment by iafan** [comment]
Please try with syntax highlighting turned off. Will caret display as expected? (Make sure you force-reload the page for CSS changes to take effect.)

**Comment by xi2** [comment]
Hi. With syntax highlighting off the caret is now displayed as expected. With syntax highlighting on the caret is not displayed.

Browser: Firefox ESR 52.2.0 (latest Debian version). OS: GNU/Linux

**Comment by PumpkinSeed** [comment]
With syntax highlighting off the caret is also displayed for me. 

Chrome 53, Fedora

**Comment by im-auld** [comment]
Turning off syntax highlighting fixes the issue for me as well:

Chrome: Version 56.0.2924.87 (64-bit)
OS: OSX 10.13.3

**Comment by iafan** [comment]
@im-auld thanks for the additional feedback. I'm glad it works for you now! Version 57 is pretty old (it's been a year already since its release; current Chrome on Mac is at version 65 already).
